app/views/usage.scala.html

```diff
@@ -203,6 +204,37 @@ <h2>Current Version(s)</h2>
                     </p>
                 </article>
                 <br/>
+                <p>
+                    <a name='completion'></a>
+                    <h2>Shell Completion</h2>
+                    <p>
+                    With SDKMAN!'s rich set of commands and subcommands, it can sometimes be hard to remember which
+                    commands are supported or which subcommands are allowed by a command.
+                    Wouldn't it be nice if the shell would autocomplete them for you?
+                    Luckily, SDKMAN! comes packaged with shell completions for both Bash and Zsh (not supported yet).
```

Comment:
Maybe we should just leave out ZSH for now until we have added completion.

Reply:
Good point!